frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -380,38 +379,37 @@ export const OperatorHubItemDetails: React.FC<OperatorHubItemDetailsProps> = ({
             </PropertiesSidePanel>
             <div className="co-catalog-page__overlay-description">
               {isAWSSTSCluster(cloudCredentials, infrastructure, authentication) &&
-                showWarn &&
+                showCSTokenWarn &&
                 infraFeatures?.find((i) => i === InfraFeatures.TokenAuth) && (
-                  <Alert
-                    isInline
-                    variant="warning"
+                  <CloudServiceTokenWarningAlert
                     title={t('olm~Cluster in STS Mode')}
-                    actionClose={<AlertActionCloseButton onClose={() => setShowWarn(false)} />}
-                    className="pf-v5-u-mb-lg"
-                  >
-                    <p>
-                      {t(
-                        'olm~This cluster is using AWS Security Token Service to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, you will need to provide a role ARN (with an attached policy) during installation. Please see the operator description for more details.',
-                      )}
-                    </p>
-                  </Alert>
+                    message={t(
+                      'olm~This cluster is using AWS Security Token Service to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, you will need to provide a role ARN (with an attached policy) during installation. Please see the operator description for more details.',
```

Comment:
Small wording suggestion
```suggestion
                      'olm~This cluster is using AWS Security Token Service to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, you must provide a role ARN (with an attached policy) during installation. Please see the operator description for more details.',
```